folsom-opencensus/src/main/java/com/spotify/folsom/opencensus/OpenCensusTracer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2019 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.spotify.folsom.opencensus;
+
+import static io.opencensus.trace.AttributeValue.stringAttributeValue;
+import static io.opencensus.trace.Span.Kind.CLIENT;
+import static java.util.Objects.requireNonNull;
+
+import com.spotify.folsom.Span;
+import com.spotify.folsom.Tracer;
+import io.opencensus.trace.Tracing;
+import java.util.concurrent.CompletionStage;
+
+class OpenCensusTracer implements Tracer {
+
+  @Override
+  public Span span(final String name, final CompletionStage<?> future, final String operation) {
+    return internalSpan(name, future, operation, null);
+  }
+
+  @Override
+  public Span span(
+      final String name,
+      final CompletionStage<?> future,
+      final String operation,
+      final String key) {
+    requireNonNull(key);
+    return internalSpan(name, future, operation, key);
+  }
+
+  @SuppressWarnings("MustBeClosedChecker")
+  private Span internalSpan(
+      final String name,
+      final CompletionStage<?> future,
+      final String operation,
+      final String key) {
+    requireNonNull(name);
+    requireNonNull(future);
+    requireNonNull(operation);
+
+    final io.opencensus.trace.Tracer tracer = Tracing.getTracer();
```

Comment:
typically this is a static variable - https://github.com/census-instrumentation/opencensus-java/blob/master/examples/src/main/java/io/opencensus/examples/trace/MultiSpansTracing.java#L31

Reply:
Ah cool, will fix.